### DISCUSSION TITLE
Share a single branch

### DISCUSSION THREAD

**Discussion by DarkwebStreamer** [community/community#37268] [New to GitHub] [answered]
Is it possible to share just a single branch of a repository with another user? Without them having access to any changes made in other branches?

I saw another post where someone mentioned there isn't a "clean" way to do this (posted below) - I'm curious about the possible workarounds, and issues they might contain.

Thank you.

link: https://github.com/orgs/community/discussions/35386

**Comment by airtower-luna** [comment, marked as answer]
No, that's not possible. Someone either has read access to a repository, or they don't.

The easiest solution is probably to create another repository and push only the branch you want to share to there. See [Git Basics - Working with Remotes](https://git-scm.com/book/en/v2/Git-Basics-Working-with-Remotes) if you're not familiar with working with multiple remote repositories yet.

**Reply by DarkwebStreamer** [reply]
Thank you!

**Reply by airtower-luna** [reply]
Glad I could help! Marking the answer would be appreciated. :slightly_smiling_face:

**Reply by DarkwebStreamer** [reply]
Will do - thanks!
